addon/array/uniq-by.js

```diff
@@ -12,6 +12,7 @@ export default createClassComputed(
       if (array === undefined || key === undefined) {
         return array;
       }
+
```

Comment:
We should probably do a `array = emberA(array);` here. That was we skip the polyfill for newer Ember version, and we are more consistent with the other array macros.

Reply:
I'm not sure what you mean tbh.

Reply:
I'll fix it in a subsequent PR and tag you.